src/components/app/details/cicdHistory/Sidebar.tsx

```diff
@@ -0,0 +1,273 @@
+                return generatePath(path, { ...rest, triggerId: id }) + '/' + currentTab
+            } else {
+                return generatePath(path, { ...rest, buildId: id }) + '/' + currentTab
+            }
+        }
+
+        return (
+            <ConditionalWrap
+                condition={Array.isArray(ciMaterials)}
+                wrap={(children) => (
+                    <TippyHeadless
+                        placement="right"
+                        interactive
+                        render={() => (
+                            <SummaryTooltipCard
+                                status={status}
+                                startedOn={startedOn}
+                                triggeredBy={triggeredBy}
+                                triggeredByEmail={triggeredByEmail}
+                                ciMaterials={ciMaterials}
+                                gitTriggers={gitTriggers}
+                            />
+                        )}
+                    >
+                        {children}
+                    </TippyHeadless>
+                )}
+            >
+                <NavLink to={getPath} className="w-100 ci-details__build-card" activeClassName="active">
+                    <div
+                        className="w-100"
+                        style={{
+                            height: '64px',
+                            display: 'grid',
+                            gridTemplateColumns: '20px 1fr',
+                            padding: '12px 0',
+                            gridColumnGap: '12px',
+                        }}
+                    >
+                        <div
+                            className={`dc__app-summary__icon icon-dim-20 ${status
+                                ?.toLocaleLowerCase()
+                                .replace(/\s+/g, '')}`}
+                        />
+                        <div className="flex column left dc__ellipsis-right">
+                            <div className="cn-9 fs-14">{moment(startedOn).format(Moment12HourFormat)}</div>
+                            <div className="flex left cn-7 fs-12">
+                                {type === HistoryComponentType.CD && (
+                                    <>
+                                        <div className="dc__capitalize">
+                                            {['pre', 'post'].includes(stage?.toLowerCase()) ? `${stage}-deploy` : stage}
+                                        </div>
+                                        <span className="dc__bullet dc__bullet--d2 ml-4 mr-4"></span>
+
+                                        {artifact && (
+                                            <div className="dc__app-commit__hash dc__app-commit__hash--no-bg">
+                                                <Docker className="commit-hash__icon grayscale" />
+                                                {artifact.split(':')[1].slice(-12)}
```

Comment:
Can we have some fail-safe around `artifact.split(':')[1]`?

Reply:
not required